packages/policy/src/changer.ts

```diff
@@ -76,8 +87,48 @@ export async function addCodeOfConduct(repo: GitHubRepo, octokit: Octokit) {
     upstreamOwner: owner,
     upstreamRepo: name,
     fork: false,
+    branch: `policy-bot-${uuid()}`,
+  });
+}
+
+/**
+ * Submit a pull request to the target repository to add a SECURITY.md.
+ * @param repo Repository name - ex: "repo-automation-bots"
+ * @param octokit Pre-authenticated octokit instance
+ */
+export async function addSecurityPolicy(repo: GitHubRepo, octokit: Octokit) {
+  // first, make sure there's no open PR for this
+  const title = 'chore: add SECURITY.md';
+  const prs = await octokit.search.issuesAndPullRequests({
+    q: `repo:${repo.full_name} "${title}" in:title is:open`,
+  });
+  if (prs.data.total_count > 0) {
+    return;
+  }
+
+  // fetch the SECURITY.md from `googleapis/.github`
+  const content = await getSecurity();
+
+  // submit the PR
+  const changes: Changes = new Map([
+    [
+      'SECURITY.md',
+      {
+        content,
+        mode: '100644',
+      },
+    ],
+  ]);
+
+  const [owner, name] = repo.full_name.split('/');
+  await createPullRequest(octokit, changes, {
+    title,
+    message: title,
+    description: 'add a security policy',
+    upstreamOwner: owner,
+    upstreamRepo: name,
+    fork: false,
     primary: repo.default_branch,
-    retry: 0,
```

Comment:
It's super important to make sure both calls to `createPullRequest` use `retry: 0` in the options

Reply:
oops. thought it was an old thing, but i needed to `npm i` when i rebased. fixed